README.md

```diff
@@ -4,87 +4,67 @@ Unbabel's PHP SDK is a wrapper around the HTTP API found at http://developers.un
 
 ## Requirements ##
 
-* PHP >= 5.3
+* PHP: 5.6 / 7.x
 
 ## Installation ##
 
-The recommended way to install is through composer.
+The recommended way to install is through composer:
 
-Just create a `composer.json` file for your project:
-
-```json
-{
-    "require": {
-        "unbabel/unbabel-php": "dev-master"
-    }
-}
+```bash
+$ composer require unbabel/unbabel-php
 ```
 
-**Tip:** browse [`unbabel/unbabel-php`](https://packagist.org/packages/unbabel/unbabel-php) page to choose a stable version to use, avoid the `dev-master` meta constraint.
-
 And run these two commands to install it:
 
 ```bash
 $ curl -sS https://getcomposer.org/installer | php
 $ composer install
 ```
 
-Now you can add the autoloader, and you will have access to the library:
+## Usage
 
 ```php
 <?php
 
 require 'vendor/autoload.php';
-```
 
-## Usage
+use Unbabel\Unbabel;
+use GuzzleHttp\Client;
+
+$httpClient = new Client();
+$unbabel = new Unbabel(
+    'username', 
+    'apiKey',
+     false, // call sandbox?
+     $httpClient
+);
+
+$opts = array('callback_url' => 'http://my-awesome-app/unbabel_callback.php');
```

Comment:
The url is invalid. The ".com" is missing. Running this example like it is will return a HTTP 400 Bad request.